styles.css

```diff
@@ -452,3 +491,358 @@ div.buttoncontact button {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+  }
+
+  div.headline,
+  div.profileimage {
+    width: 50%;
+    margin: 1em auto;
+    justify-content: center;
+    align-self: center;
+  }
+
+  div.headline :nth-child(1) {
+    margin-left: 3em;
+    line-height: 1em;
+    font-size: 3rem;
+  }
+
+  div.headline :nth-child(2) {
+    margin-left: 3.5em;
+  }
+
+  div.headline :nth-child(3) {
+    width: 60%;
+    margin-left: 7em;
+  }
+
+  div.invitecontactme p {
+    width: 100%;
+    margin: 0 2em;
+    text-align: center;
+  }
+
+  div.profileimage > div.boxcode p {
+    font-size: 0.9rem;
+  }
+
+  div.photolayer1,
+  div.photolayer2,
+  div.photolayer3 {
+    width: 20em;
+    height: 20em;
+  }
+
+  div.desknavbar { /* ******** bottom navbar ********* */
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    position: fixed;
+    width: 100%;
+    top: 92vh;
+    z-index: 3;
+    background-color: var(--dark-bg);
+  }
+
+  div.desknavbar > div.logo {
+    margin-left: 10em;
+  }
+
+  div.navbar2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    width: 50%;
+  }
+
+  div.navbar2 ul {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    padding: 0;
+    width: 75%;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  div.navbar2 ul li a {
+    text-decoration: none;
+    font-family: 'Inconsolata', 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 1.25rem;
+    line-height: 1.3125rem;
+    text-transform: capitalize;
+    color: var(--text);
+  }
+
+  div.navbar2 ul li a:hover {
+    color: var(--mark-text2);
+  }
+
+  div.aboutme { /* ******** about me with icon ********** */
+    width: 80%;
+    margin: auto;
+  }
+
+  div.myskills { /* ********** my skills section *********** */
+    max-width: 1500px;
+    width: 100%;
+  }
+
+  div.skillcards {
+    flex-wrap: nowrap;
+    flex-grow: 0;
+    width: 80%;
+  }
+
+  div.skill {
+    width: 11%;
+  }
+
+  div.works {
+    flex-direction: row;
+    width: 80%;
+  }
+
+  div.card {
+    margin: 1em;
+  }
+
+  footer { /* ********** footer contacts ********** */
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    margin-bottom: 5em;
+  }
+
+  div.footerleft,
+  div.footeright {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+  }
+
+  div.invitecontactme {
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  div.socialnet { /* *********** social networks ********** */
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  div.socialnet ul {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 80%;
+  }
+
+  div.logo.contact {
+    margin-top: 1.7em;
+  }
+
+  div.boxcontact {
+    width: 100%;
```

Comment:
## in the footer I can see that it is located differently in the original design I can see that the network contacts section is in a row with the presentation section
![Captura de pantalla (102)](https://user-images.githubusercontent.com/119135868/228362542-134d829d-628d-473b-a3cd-d5862df65beb.png)
## your design has this distribution

![Captura de pantalla (104)](https://user-images.githubusercontent.com/119135868/228362927-8d2e3846-1005-4e4e-a2a5-474080ab9962.png)
### I suggest that you reduce the width of the different sections so that you can see the 3 in a row so that it looks more like the original
![Captura de pantalla (105)](https://user-images.githubusercontent.com/119135868/228364010-45b3d496-a3fc-4093-ab5b-e434894a69e0.png)